manifests/config/passenger.pp

```diff
@@ -175,6 +180,67 @@
         custom_fragment         => template('foreman/_assets.conf.erb', 'foreman/_ssl_virt_host_include.erb',
                                             'foreman/_suburi.conf.erb'),
       }
+
+      $pki_dir = '/etc/pki/foreman'
```

Comment:
I think we should prefer distro locations if possible, but fall back to something generic.